TrafficCapture/dockerSolution/src/main/docker/migrationConsole/workflows/templates/clusterTemplates.yaml

```diff
@@ -6,6 +6,413 @@ spec:
   serviceAccountName: argo-workflow-executor
   
   templates:
+    - name: elasticsearch-1-5-single-node
+      inputs:
+        parameters:
+          - name: cluster-name
+          - name: namespace
+      container:
+        image: dtzar/helm-kubectl:latest
+        command: [ sh, -c ]
+        args:
+          - |
+            set -euo pipefail
```

Comment:
See my other comments for the 2.4 template, they should also apply to this template as well